### ISSUE TITLE
ci view: short or brief output?

### ISSUE THREAD

**Issue by claytonrcarter** [zaquestion/lab#261]
Not sure if it's just my use case or if it would be useful for others, but I often just want to get a quick gist of what CI is currently up to. `ci view` provides this of course, but if I don't want to trace or interact with jobs, it feels like overkill, plus I have to `q` out of it. What I would love is something like `lab ci status` or maybe `lab ci view --brief` that just lists the job names and their current status. For example (where I only have one job named "test"):
```
$ git push
...
$ lab ci status
test: pending
$ lab ci status
test: running
$ lab ci status
test: success
```

**Comment by zaquestion** [comment]
:+1: to a `lab ci status` I've wanted one for a while and something like the output you're describing is appealing. It seems like the biggest challenge dealing with the fact that pipelines have multiple jobs. So if you're proposing just doing a textual semi-ordered list that seems reasonable
```
$ lab ci status
build: success
test:  running
deploy: pending
```
Actually sounds pretty rad!

**Comment by nkprince007** [comment]
I totally second this approach. (adding a few emojis makes it even cute though)

```
$ lab ci status
✅ build: success
🏃🏻‍♂️ test: running
🕐 deploy: pending
❌ e2e: failed
⏸ docker-build: paused
```

**Comment by MartinDelille** [comment]
I posted a proposition here: https://github.com/zaquestion/lab/issues/259#issuecomment-447343828

**Comment by claytonrcarter** [comment]
Closed (a while ago) by #273
